Swiften/Parser/PayloadParsers/UnitTest/ClientStateParserTest.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2017 Tarun Gupta
+ * Licensed under the simplified BSD license.
+ * See Documentation/Licenses/BSD-simplified.txt for more information.
+ */
+
+#include <cppunit/extensions/HelperMacros.h>
```

Comment:
We are trying to slowly transition away from CppUnit tests to GoogleTest. They are much less verbose to write, so your actual test cases are more visible. See https://github.com/swift/swift/blob/master/Swiften/Base/UnitTest/ByteArrayTest.cpp or https://github.com/swift/swift/blob/master/Swift/Controllers/Chat/UnitTest/ChatMessageParserTest.cpp for examples.